src/eigenlayer/interfaces/IZKMRStakeRegistry.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import {IStrategy} from
+    "eigenlayer-contracts/src/contracts/interfaces/IStrategy.sol";
+import {ISignatureUtils} from
+    "eigenlayer-contracts/src/contracts/interfaces/ISignatureUtils.sol";
+
+/// @notice An Eigenlayer strategy and weight multiplier
+/// @dev A strategy is a contract that represent an underlying staked asset (ERC20)
+struct StrategyParams {
+    IStrategy strategy;
+    uint96 multiplier;
+}
+
+/// @notice A `quorum` of Eigenlayer strategies and weight multipliers
+/// @dev An array of strategy parameters define the quorum
+struct Quorum {
+    StrategyParams[] strategies;
+}
+
+/// @notice A point on an elliptic curve
+/// @dev Used to represent an ECDSA public key
+struct G1Point {
```

Comment:
Remove the G1 - for ecdsa, it's not a pairing friendly curve (i.e. it can't do BLS signatures) so there is no G1 or G2 or GT point distinctions, they all live on the same and unique curve. So just Point is good :)

Reply:
I'm just going to call it `PublicKey`